API/hermes/hermes.h

```diff
@@ -214,6 +215,11 @@ class HERMES_EXPORT HermesRuntime : public jsi::Runtime {
       const std::shared_ptr<const jsi::Buffer> &sourceMapBuf,
       const std::string &sourceURL);
 
+  /// Returns the underlying low level Hermes VM runtime instance.
+  /// This should be avoided as the lower level APIs are unsafe
+  /// and they can change without notice.
```

Comment:
Strengthening the wording here a little
```suggestion
  /// This function is considered unsafe and unstable.
  /// Direct use of a vm::Runtime should be avoided as the lower level APIs are unsafe
  /// and they can change without notice.
```